packages/medusa/src/services/customer.ts

```diff
@@ -139,30 +137,28 @@ class CustomerService extends TransactionBaseService<CustomerService> {
       order: { created_at: "DESC" },
     }
   ): Promise<[Customer[], number]> {
-    return await this.atomicPhase_(async (manager) => {
-      const customerRepo = manager.getCustomRepository(this.customerRepository_)
+    const manager = this.manager_
+    const customerRepo = manager.getCustomRepository(this.customerRepository_)
 
-      let q
-      if ("q" in selector) {
-        q = selector.q
-        delete selector.q
-      }
+    let q
+    if ("q" in selector) {
+      q = selector.q
+      delete selector.q
+    }
 
-      const query = buildQuery<Selector<Customer>, Customer>(selector, config)
+    const query = buildQuery<Selector<Customer>, Customer>(selector, config)
 
-      return await customerRepo.listAndCount(query, q)
-    })
+    return await customerRepo.listAndCount(query, q)
   }
 
   /**
    * Return the total number of documents in database
    * @return {Promise} the result of the count operation
    */
   async count(): Promise<number> {
-    return await this.atomicPhase_(async (manager) => {
-      const customerRepo = manager.getCustomRepository(this.customerRepository_)
-      return await customerRepo.count({})
-    })
+    const manager = this.manager_
+    const customerRepo = manager.getCustomRepository(this.customerRepository_)
+    return await customerRepo.count({})
```

Comment:
question(non blocking): why are we passing an empty object here? the options are optional and I would have assume that no arguments would give the same result no?